upgrade/1.2/Stage_1.md

```diff
@@ -8,27 +8,29 @@
 
     > NOTE: You may need to reset the root password for each node after it is rebooted
 
-    **IMPORTANT:** You may encounter an error like shown below.  If this is the case, just re-run the same command for the node upgrade and it will pick up at that point and conitune.
+    **Known Issues:**
+    * It is possible to encounter an error like shown below.  If this is the case, just re-run the same command for the node upgrade and it will pick up at that point and conitune.
 
-    ```text
-    ====> REDEPLOY_CEPH ...
-    /usr/bin/ssh-copy-id: INFO: Source of key(s) to be installed: "/root/ceph.pub"Number of key(s) added: 1 Now try logging into the machine, with:   "ssh 'root@ncn-s003'"
-    and check to make sure that only the key(s) you wanted were added.Error EINVAL: Traceback (most recent call last):
-    ```
+      ```text
+      ====> REDEPLOY_CEPH ...
+      /usr/bin/ssh-copy-id: INFO: Source of key(s) to be installed: "/root/ceph.pub"Number of key(s) added: 1Now try logging into the machine, with:   "ssh 'root@ncn-s003'"
+      and check to make sure that only the key(s) you wanted were added.Error EINVAL: Traceback (most recent call last):
+      ```
+    * During the storage node rebuild, it is possible that ceph health may report ***HEALTH_WARN 1 daemons have recently crashed***.  This can occur occasionally as part of the shutdown process of the node being rebuilt.  Please refer to the [Dump Ceph Crash Data](operations/../../../operations/utility_storage/Dump_Ceph_Crash_Data.md) located in the Operations/Utility Storage section of the guide.
```

Comment:
```suggestion
    * During the storage node rebuild, it is possible that Ceph health may report ***HEALTH_WARN 1 daemons have recently crashed***.  This can occur occasionally as part of the shutdown process of the node being rebuilt. Please refer to the [Dump Ceph Crash Data](operations/../../../operations/utility_storage/Dump_Ceph_Crash_Data.md) procedure for more information.
```